datadog/resource_datadog_service_level_objective_test.go

```diff
@@ -0,0 +1,199 @@
+package datadog
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/zorkian/go-datadog-api"
+)
+
+// config
+const testAccCheckDatadogServiceLevelObjectiveConfig = `
+resource "datadog_service_level_objective" "foo" {
+  name = "name for metric SLO foo"
+  type = "metric"
+  description = "some description about foo SLO"
+  query = {
+	numerator = "sum:my.metric{type:good}.as_count()"
+	denominator = "sum:my.metric{*}.as_count()"
+  }
+
+  thresholds = [
```

Comment:
Terraform 0.12 doesn't seem to support this syntax. It looks like TF 0.11 (at least 0.11.10) + 0.12 allow you to use 

```
thresholds {
}

thresholds {
}
```

Could we use that syntax to support more? (Also in the docs)